skyplane/planner/planner.py

```diff
@@ -46,49 +44,31 @@ def __init__(self, transfer_config: TransferConfig):
             with gcp_quota_path.open("r") as f:
                 self.quota_limits["gcp"] = json.load(f)
 
+        # Loading the vcpu information - a dictionary of dictionaries
+        # {"cloud_provider": {"instance_name": vcpu_cost}}
+        self.vcpu_info = defaultdict(dict)
+        with path("skyplane.data", "vcpu_info.csv") as file_path:
+            with open(file_path, "r") as csvfile:
+                reader = csv.reader(csvfile)
+                next(reader)  # Skip the header row
+
+                for row in reader:
+                    instance_name, cloud_provider, vcpu_cost = row
+                    vcpu_cost = int(vcpu_cost)
+                    self.vcpu_info[cloud_provider][instance_name] = vcpu_cost
+
     def plan(self) -> TopologyPlan:
         raise NotImplementedError
 
-    @staticmethod
-    def _vm_to_vcpus(cloud_provider: str, vm: str) -> int:
-        if cloud_provider == "aws":
-            n_vcpus = re.findall(r"\d*x", vm)[0]
-            return 4 * int(n_vcpus.replace("x", "") or 1) if "x" in n_vcpus else int(n_vcpus)
-        elif cloud_provider == "azure":
-            n_vcpus = re.findall(r"\d+", vm)
-            return int(n_vcpus[0])
-        else:
-            return int(vm.split("-")[-1])
-
-    @staticmethod
-    def _vcpus_to_vm(cloud_provider: str, vcpus: int) -> str:
-        if cloud_provider == "aws":
-            return "m5.large" if vcpus == 2 else "m5.xlarge" if vcpus == 4 else f"m5.{vcpus // 4}xlarge"
-        vm_family = "Standard_D{}_v5" if cloud_provider == "azure" else "n2-standard-{}"
-        return vm_family.format(vcpus)
-
-    # @staticmethod
```

Comment:
I saved this function somewhere else in my notes instead of letting it hanging like this since we are not using it anyways